src/erdstall.ts

```diff
@@ -62,33 +65,62 @@ export interface Leaver extends Exiter, Withdrawer {
 	leave(): Promise<Stages<Promise<ethers.ContractTransaction>>>;
 }
 
-export interface Subscriber {
+export interface BoundSubscriber {
 	subscribe(): Promise<void>;
+}
```

Comment:
I'd set this to
```suggestion
export interface BoundSubscriber {
	subscribeSelf(): Promise<void>;
}
```
and let the `Subscriber` interface be `subscribe(who?: Address): Promise<void>`. Makes things simpler.